Libraries/Components/Button.js

```diff
@@ -154,6 +155,40 @@ type ButtonProps = $ReadOnly<{|
    * For arrow keys, add "ArrowLeft", "ArrowRight", "ArrowUp", "ArrowDown",
    */
   validKeysUp?: ?Array<string>,
+  
+  /**
+   * Specifies whether the view can become key view
+   */
+  canBecomeKeyView?: ?boolean, // TODO(macOS ISS#2323203)
+
+  /**
+   * A Boolean value indicating whether the view needs its panel to become the 
+   * key window before it can handle keyboard input and navigation.
+   */
+  needsPanelToBecomeKey?: ?boolean, // TODO(macOS ISS#2323203)
+  
+  /**
+  * The view object that follows the current view in the key view loop.
+  */
+  nextKeyView?: ?RCTView, // TODO(macOS ISS#2323203)
+  
+  /**
+  * The closest view object in the key view loop that follows the current 
+  * view in the key view loop and accepts first responder status.
+  */
+  nextValidKeyView?: ?RCTView, // TODO(macOS ISS#2323203)
```

Comment:
This is usually readonly on the native side. I'm not sure in which cases you'd want to set this separately from `nextKeyView`.

Reply:
Good catch, we might want to remove this. I'm investigating if we actually do need this for anything

Reply:
Oh yeah, also @appden nice to meet you! I work with Chris and I should be taking over this PR to get it checked in. Just an FYI =)

Reply:
Good point- I think the readonly props should stay readonly